plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/cdk/server/core/internal/listeners/ServiceManagerUtility.java

```diff
@@ -17,17 +17,17 @@
 	public static HashMap<String, String> parseLines(String[] lines) {
 		HashMap<String, String> adbEnv = new HashMap<>();
 
-		String setEnvWin = "setx ";
```

Comment:
I'm very confused why you're making this patch. First, this class is used for both cdk 2.x and 3.x, so it needs to work for both. I know for a fact cdk 2.x on windows with certain plugin versions will still return the setx syntax, so we cannot remove parsing ability for that. 

Do you have any confirmation it has switched to SET from setx?  Is this based on what shell you are using? 

I think this patch will likely break things.